firmware/application/apps/ui_debug.cpp

```diff
@@ -512,7 +427,7 @@ void DebugMenuView::on_populate() {
         {"Pers. Memory", ui::Theme::getInstance()->fg_darkcyan->foreground, &bitmap_icon_memory, [this]() { nav_.push<DebugPmemView>(); }},
         //{ "Radio State",	ui::Theme::getInstance()->bg_darkest->foreground,	nullptr,	[this](){ nav_.push<NotImplementedView>(); } },
```

Comment:
while you're at it, please remove that commented out entry too

Reply:
All others too? Ahh ok

Reply:
I removed the Memory Usage entry, not sure if should remove the Radio State view as well. it looks like not finished instead of moved off external

how do you think or You have other info?

Reply:
If it's commented out I would remove it.
I'm tired of dead code in comments everywhere.

Reply:
ok sure i will remove it too